### PR TITLE
New package: argtable

### PR DESCRIPTION
**Pull request by Magister-Bergerac**

### CODE REVIEW

**Comment by jnbr** [comment]
Since this package provides a shared library, an entry in `common/shlibs` is needed: 
`=> WARNING: argtable-2.13_1: libargtable2.so.0 not found in common/shlibs!`

Also, files only needed for development should go into a `argtable-devel` subpackage.

**Comment by maxice8** [comment]
> Since this package provides a shared library, an entry in common/shlibs is needed:

Not really, lots of packages that provide libs have no shlib entry, it is most common to add only when a package makes use of it.

**Comment by jnbr** [comment]
> Not really, lots of packages that provide libs have no shlib entry, it is most common to add only when a package makes use of it.
Thank you, wasn't aware of that.
Sorry @Magister-Bergerac for wrong information.

**Comment by maxice8** [comment]
If you're going to add shlib then make it in the same commit as the one you introduced the package, also clustalo seems unrelated to argtable, make a separate PR for it if you plan on adding it.

**Comment by Magister-Bergerac** [comment]
How do I add it in the same commit?

**Comment by Vaelatern** [comment]
You amend commits. Or fix them up. Google it.

File: common/shlibs [hunk 1]
```diff
@@ -16,6 +16,7 @@
 # PLEASE NOTE: when multiple packages provide the same SONAME, the first
 # one (order top->bottom) is preferred over the next ones.
 #
+libargtable2.so.0 glibc-2.25_1
```
Comment (Vaelatern) [inline common/shlibs]: NO.
NO.
NO.

File: srcpkgs/argtable/template [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Template file for 'argtable'
+pkgname=argtable
+version=2.13
+revision=1
+wrksrc="argtable2-13"
```
Comment (Vaelatern) [inline srcpkgs/argtable/template]: `wrksrc="${pkgname}${version//./-}"`

File: common/shlibs (hunk 1 above)
Comment (Vaelatern) [reply common/shlibs]: No.
Remove the change completely.

**Comment by Vaelatern** [comment]
Shlibs still broken. I'm not closing the PR, but I'm not merging it.

**Comment by Vaelatern** [comment]
Please clean the commit history. You need to have one commit.
That or I can squash and merge, but I won't with the current `short_desc`. Understand why as follows:
```
[-] argtable-2.13_1                 Argtable is an ANSI C command line parser
```
So it should be `ANSI C command line parser`

**Pull request closed without merging**